mozilla_vpn/explores/events.explore.lkml

```diff
@@ -0,0 +1,9 @@
+include: "//looker-hub/mozilla_vpn/views/events.view.lkml"
+
+explore: events {
```

Comment:
The existing events explore is actually [named `event_counts`](https://github.com/mozilla/looker-hub/blob/d30493f80499b85f466d04bbb470ae3a58256251/mozilla_vpn/explores/events.explore.lkml#L9), so I think the idea would be to [refine](https://cloud.google.com/looker/docs/lookml-refinements) that explore (by using the same name with a `+` prefix), rather than create a new explore named `events` (which is what this currently does).
```suggestion
include: "//looker-hub/mozilla_vpn/explores/events.explore.lkml"

explore: +event_counts {
```